sdks/java/container/build.gradle

```diff
@@ -60,8 +60,11 @@ golang {
 }
 
 docker {
-  // TODO(herohde): make the name easier to generate for releases.
-  name System.properties['user.name'] + '-docker-apache.bintray.io/beam/java:latest'
+  def repositoryRoot = System.properties["docker-repository-root"]
+  if (!repositoryRoot) {
+    repositoryRoot = "${System.properties["user.name"]}-docker-apache.bintray.io/beam"
```

Comment:
Your trying to stick double quotes inside a double quoted string.

Reply:
This works because it's being interpreted as groovy code (inside of the `${}`).

Reply:
mind blown.